<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>981</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>COURSE/PROGRAM ADVISING SLIP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Student's Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Student Number:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Program/Year:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>100</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStudentName">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStudentNumber">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProgramYear">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDate">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>1261</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Nature of Advising</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="checkBox1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Thesis/Design Subject Concerns</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>441</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mentoring/Clarification on the Topic
of the subjects Enrolled</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>441</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Requirements in Courses Enrolled</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox4">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>60</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Concerns about Electives/Tracks
in the Curriculum</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox5">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>100</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Concerns on Internship/OJT Matters</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>140</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Concerns regarding Placement/Employment
Opportunities</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox7">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>190</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Concerns regarding Personal/Family, etc.</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox8">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>220</y>
       <width>551</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Others: (Please Specify)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditConcern">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>270</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonTakePicture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSubmit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>660</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Academic Adviser:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAcademicAdviser">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>PAMBID, LUIS DANIEL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelPicture">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRefresh">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditStudentName</tabstop>
  <tabstop>lineEditStudentNumber</tabstop>
  <tabstop>lineEditProgramYear</tabstop>
  <tabstop>comboBoxAcademicAdviser</tabstop>
  <tabstop>lineEditDate</tabstop>
  <tabstop>pushButtonTakePicture</tabstop>
  <tabstop>pushButtonSubmit</tabstop>
  <tabstop>checkBox1</tabstop>
  <tabstop>checkBox2</tabstop>
  <tabstop>checkBox3</tabstop>
  <tabstop>checkBox4</tabstop>
  <tabstop>checkBox5</tabstop>
  <tabstop>checkBox6</tabstop>
  <tabstop>checkBox7</tabstop>
  <tabstop>checkBox8</tabstop>
  <tabstop>lineEditConcern</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
